sklearn/utils/multiclass.py

```diff
@@ -157,6 +157,24 @@ def is_multilabel(y):
         return len(labels) < 3 and (y.dtype.kind in 'biu' or  # bool, int, uint
                                     _is_integral_float(labels))
 
+def check_non_regression_targets(y):
+    """Determine if target `y` is of non regression type
+
+    For the moment, the following (as defined in type_of_target) are allowed:
+        - 'binary', 'multiclass', 'multiclass-multioutput', 
+        'multilabel-indicator', 'multilabel-sequences'
+        - 'continuous', if only there are only two unique values
+
+    Parameters
+    ----------
+    y : array-like
+    """
+    y_type = type_of_target(y)
+    if y_type not in ['binary', 'multiclass', 'multiclass-multioutput', 
+            'multilabel-indicator', 'multilabel-sequences']:
+        raise ValueError("Unknown label type: %r" % y)
+
+
 
```

Comment:
PEP8: only 2 blank lines to separate top level blocks (e.g. class or function definitions).